oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextIndexEditorContext.java

```diff
@@ -165,9 +164,8 @@ public void closeWriter() throws IOException {
       NodeBuilder status = definitionBuilder.child(IndexDefinition.STATUS_NODE);
       status.setProperty(IndexDefinition.STATUS_LAST_UPDATED, getUpdatedTime(currentTime), Type.DATE);
       status.setProperty("indexedNodes", indexedNodes);
-      if (storedIndexDefinitionEnabled() && reindex) {
-        NodeBuilder indexDefinition = definitionBuilder.child(STATUS_NODE);
-        indexDefinition.setProperty(IndexDefinition.REINDEX_COMPLETION_TIMESTAMP, ISO8601.format(currentTime), Type.DATE);
+      if (reindex) {
```

Comment:
the reindex completion timestamp is exposed in the index stats. I could not find a reason why this was not set for NON stored index defs. I don't see issues in setting it in any case. Feedback is welcome @thomasmueller @nit0906 @averma21

Reply:
I don't know, but I'm quite sure @tihom88  would know; could we ask him? (I know he is on PTO, but maybe we could still ask him)